code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1898,22 +1898,22 @@
 	icon_state = "markings_tiger"
 
 /datum/sprite_accessory/body_markings/tigerhead
-	name = "Tiger Body + Head"
+	name = "Tiger Body and Head"

```

Comment:
I had to change these marking names because Nano hated the plus signs and it meant that the buttons for each of these styles wouldn't work in the interfaces. Getting rid of the plus signs fixed it.